js/views/startScreen.js

```diff
@@ -124,7 +129,7 @@ var StartScreen = React.createClass({
       // Small Promo Image configuration
       posterStyle.backgroundSize = "auto";
       return (
-        <div className="startScreen" onClick={this.handleClick} onTouchEnd={this.handleClick} style={screenStyle.style}>
+        <div className="startScreen" onClick={this.handleClick} onTouchEnd={this.handleClick} onMouseDown={this.handleMouseDown} style={screenStyle.style}>
```

Comment:
@mistermendez, the problem in PBW-3656 is that when you hold down the mouse button for a while, the cursor changes to 'text selection' cursor. To prevent it, onMouseDown is used here.